README.md

```diff
@@ -656,7 +656,7 @@ the following:
 - `insertOnly`: `boolean` will prevent any properties from persisting if the job already exists. Defaults to false.
 
 ```js
-job.unique({'data.type': 'active', 'data.userId': '123', nextRunAt(date)});
+job.unique({'data.type': 'active', 'data.userId': '123'});
```

Comment:
Not sure if this could be replaced with something else, but the current syntax was invalid and I'm not sure what purpose a unique constraint on `nextRunAt` would serve.

Reply:
@mmiller42 : I believe you are right. But should we have it as a separate PR?

Reply:
removed